health-services/project-factory/src/server/utils/genericUtils.ts

```
@@ -544,7 +544,7 @@ async function getSchemaBasedOnSource(request: any, isSourceMicroplan: boolean,
 
 async function createFacilitySheet(request: any, allFacilities: any[], localizationMap?: { [key: string]: string }) {
   const responseFromCampaignSearch = await getCampaignSearchResponse(request);
-  const isSourceMicroplan = checkIfSourceIsMicroplan(responseFromCampaignSearch?.CampaignDetails?.[0]);
+  const isSourceMicroplan = await isMicroplanCampaignId(request?.query?.campaignId);
   request.body.isSourceMicroplan = isSourceMicroplan;
   let schema: any = await getSchemaBasedOnSource(request, isSourceMicroplan, responseFromCampaignSearch?.CampaignDetails?.[0]?.additionalDetails?.resourceDistributionStrategy);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'schema' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)